usr/sbin/orjail

```diff
@@ -651,14 +651,15 @@ fi
 NOSETUPERROR=y
 # use firejail as security container
 if [ $USEFIREJAIL = y ]; then
+  BINARYPATH="$(readlink -f $(which "$@"))"
```

Comment:
This is not good enough. For instance: `readlink -f $(which ls)` returns an error.